src/Cohesity.Powershell/Scripts/Utility/Generate-CohesityPSCmdletTemplate.ps1

```diff
@@ -0,0 +1,167 @@
+function Generate-CohesityPSCmdletTemplate {
+   <#
+	    .SYNOPSIS
+		Creates a powershell cmdlet template. 
+		.DESCRIPTION
+		Creates a powershell cmdlet's script template based on the given method type and feature. In case if the user doesn't provide the file path, it will create a file in the current directory.
+		.LINK
+		https://cohesity.github.io/cohesity-powershell-module/#/README
+		.EXAMPLE
+		Generate-CohesityPSCmdletTemplate -MethodType <string[GET/PUT/POST/DELETE/REGISTER]> -Feature <string>
```

Comment:
what's REGISTER ?

Reply:
A cmdlet with protection source registration would use REGISTER

Reply:
Oh ok, there's some confusion, it'd mean HTTP method, can you change it to field type to ActionType ?

Reply:
Fixed